packages/blockchain-wallet-v4-frontend/src/modals/Xlm/SendXlm/FirstStep/template.success.js

```diff
@@ -181,11 +181,11 @@ const FirstStep = props => {
                 />
                 {enableToggle ? (
                   !toToggled ? (
-                    <AddressButton onClick={handleToToggle}>
+                    <AddressButton onClick={() => handleToToggle()}>
```

Comment:
because handleToToggle takes in a value it was being passed the onClick event, so the toggle never actually toggled because it looks for any val, the value was the event.

Reply:
See here: https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/931d572264209abff2778078bc24072b34f8f0f8/packages/blockchain-wallet-v4-frontend/src/modals/Xlm/SendXlm/FirstStep/index.js#L12-L15

Reply:
Ah, indeed. I did not look at the `handleToToggle` definition